lib/versioning/loose/index.spec.ts

```diff
@@ -44,6 +44,10 @@ describe('versioning/loose/index', () => {
     ${'2.4.beta'} | ${'2.4.alpha'} | ${true}
```

Comment:
> And we use a start version of `5.3`, so it looks like we should treat longer versions greater that shorter again.

But it seems like we already do (see the first test in this table)

@viceice

Reply:
🤔 here is the code which somehow fails

https://github.com/renovatebot/internal-tools/blob/716a319991bfbe27c5169dd9faed38af0d7634d0/src/commands/docker/builder.ts#L48

Reply:
I'll try to debug that tomorrow from there. Maybe I can the find the issue easier.